spec/requests/api/v1.0/service_plans_spec.rb

```diff
@@ -45,6 +45,14 @@
         expect(db_plans.count).to eq json.count
         expect(db_plans.first.portfolio_item_id).to eq portfolio_item_without_service_plan.id
       end
+
+      it "returns modified as true" do
+        expect(json.first['modified']).to be_truthy
+      end
+
+      it "returns the base schema" do
+        expect(json.first['create_json_schema']).to eq service_plan.base
+      end
```

Comment:
This seems weird to me still, it doesn't make sense that we're returning the base schema if "modified" is true. Do we need to change the structure of how we're handling this somehow, or am I thinking about this wrong?

Reply:
reworked the specs a bit to display better what we're trying to test.